pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/UniformLoadShedder.java

```diff
@@ -137,7 +137,7 @@ public Multimap<String, String> findBundlesForUnloading(final LoadData loadData,
                         brokersData.get(msgRateOverloadedBroker.getValue()).getLocalData();
                 if (overloadedBrokerData.getBundles().size() > 1
                         && (msgRateRequiredFromUnloadedBundles.getValue() >= conf.getMinUnloadMessage())) {
-                    // Sort bundles by throughput, then pick the bundle which can help to reduce load uniformly with
+                    // Sort bundles by msgRate, then pick the bundle which can help to reduce load uniformly with
```

Comment:
I think it‘s no need to fix here

Reply:
Oh, this is not to fix typos, but because the content in the comments is different from the actual logic. Here they are sorted according to msgRate. Here I fix it.

Reply:
Ah, yes, I see the wrong `sorted` method at line 189